published-namespaces.txt

```diff
@@ -296,7 +301,7 @@ AndroidX.Wear.WatchFace
 AndroidX.Wear.WatchFace.Client
 AndroidX.Wear.WatchFace.Complications
 AndroidX.Wear.WatchFace.Complications.Data
-AndroidX.Wear.WatchFace.Complications.Datasource
+AndroidX.Wear.WatchFace.Complications.DataSource
```

Comment:
Whoops, we can't change an existing namespace, we aren't going to go through the `AndroidX.Core.View.Accessibiity` nightmare again.  😁 

I think we can just remove this line, as we removed the new `Compose` namespace(s) that needed it:

```
<AndroidNamespaceReplacement Include='datasource' Replacement='DataSource' />
```

Alternatively, we can leave that and use metadata to change just this namespace back to `Datasource`.

Reply:
Added initial Metadata (there was no metadata at all) with package to namespace mapping.

Hopefully Metadata is applied after `AndroidNamespaceReplacement` MSBuild task.

Reply:
The metadata did not work because it is set to `managedReturn` instead of `managedName`.